connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceTask.java

```diff
@@ -100,7 +100,7 @@ public void start(Map<String, String> props) {
         replicationPolicy = config.replicationPolicy();
         partitionStates = new HashMap<>();
         offsetSyncsTopic = config.offsetSyncsTopic();
-        consumer = MirrorUtils.newConsumer(config.sourceConsumerConfig());
+        consumer = MirrorUtils.newConsumer(config.sourceConsumerConfig("data-consumer"));
```

Comment:
Nit: "replication" is more specific than "data" (every consumer could be thought of as one that reads "data")